hibernate-ogm/playsport-demo/src/main/java/org/hibernate/ogm/demo/playsport/core/entity/Club.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+/**
+ * Created by fabio on 20/08/2017.
+ */
+package org.hibernate.ogm.demo.playsport.core.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.search.annotations.Analyze;
+import org.hibernate.search.annotations.Field;
+import org.hibernate.search.annotations.Indexed;
+import org.hibernate.validator.constraints.NotEmpty;
+
+@Entity
+@Indexed
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler", "athletes", "employees"})
+public class Club {
```

Comment:
When a rest service retrive a list of entities from a repositories, the objects are already detached (because transaction boundaries are scoped with service methods), but I have in any case the proxies of objects. On proxy have always the property hibernateLazyInitializer. I probably miss something.

Reply:
Thanks, I didn't know these properties were going to be exposes to the REST API.